### PR TITLE
feat(chat-e2e): select folder modal tests

### PR DESCRIPTION
**Pull request by irinakartun**
**Description:**

implemented select folder modal tests

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [ ] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1538.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1538.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1538.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat-e2e/src/ui/webElements/editInputActions.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { BaseElement } from '@/src/ui/webElements/baseElement';
+import { Locator, Page } from '@playwright/test';
+
+export class EditInputActions extends BaseElement {
+  constructor(page: Page, parentLocator: Locator, selector: string) {
+    super(page, `${selector} >> ${EditSelectors.actionButton}`, parentLocator);
+  }
+
+  public async clickTickButton() {
+    await this.getNthElement(1).click();
```
Comment (IlyaBondar) [inline apps/chat-e2e/src/ui/webElements/editInputActions.ts]: I suppose it's better to use specific `data-qa` instead here and below
image

**Review by IlyaBondar — DISMISSED** [review]

File: apps/chat-e2e/src/ui/webElements/editInputActions.ts (hunk 1 above)
Comment (irinakartun) [reply apps/chat-e2e/src/ui/webElements/editInputActions.ts]: good point, thank you! Fixed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1538.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**